languages/rust/exercises/concept/structs/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+# Instructions
+
+In this exercise you'll be implementing methods on a `struct`. 
+
+You'll start with some stubbed functions in an `impl` block as well as the following `struct`:
+
+```rust
+#[derive(Clone, Debug)]
```

Comment:
Is the `derive` macro something students will be familiar with? As the student will get provided with the struct beforehand, this may not matter much though.

Reply:
This may or may not be the student's first exposure to `derive`; it's used mainly on structs and enums, and I don't think there's a dependency ordering between those concepts. That said, given that it's provided for the student, I think it's probably safe to just explain it later.

Reply:
Okay, that's fine with me. Just wanted to mention it.

Reply:
@coriolinus Is there something to be said about removing the `derive` line from the struct definition in this exercise if it's not actually needed to implement and test the exercise? I could see it as something that distracts students from the point of the exercise, especially if they've never seen it before.

Reply:
Yes, but there's also something to be said for exposing students early to this very common annotation.

I'd be happy either way. As you say, it's not required to complete the exercise, so possibly it's better to remove it here and explicitly introduce derives sometime later. (When, though? It's deeply tangled in with the general concept of traits, but deriving a trait you wrote is a particularly advanced topic.)

Reply:
I feel like it's a bit too early at this stage in order to expose students to traits, so I'm going to leave it off for now. I think it does make a little more sense to introduce them when traits come around.

Reply:
I agree with teaching it later. Its presence will start acquainting students but the full explanation can wait. From a cursory read, derived traits are introduced in [Chapter 5, section 2 of TRPL](https://doc.rust-lang.org/book/ch05-02-example-structs.html#adding-useful-functionality-with-derived-traits).

Reply:
@efx so you’re saying we _should_ leave the `derive` in but just not say anything about it? Or do you mean you’re fine with the `derive` being removed entirely from this exercise?

Reply:
> I'd be happy either way. As you say, it's not required to complete the exercise, so possibly it's better to remove it here and explicitly introduce derives sometime later

> I feel like it's a bit too early at this stage in order to expose students to traits, so I'm going to leave it off for now.

This sounds like a good plan.

Reply:
@seanchen1991 sorry; I am in favor of the first option. I've not caught up on the discussion so I could be off here.